### ISSUE TITLE
Network: Added "New Group" for a node doesn't show up until restart

### ISSUE THREAD

**Issue by NyvenZA** [RetroShare/RetroShare#2033]
Visual bug. 
The Groups are added but you can't see them on the Network page untl you restart RS. 

I added new nodes that I want in separate groups. 
* I right click on the node 
* Select Groups 
* Create New Group 
* Enter the New name. 
Then I select another node and want to add it to the same group. 
This works. 

In the Network View I do not see the new Groups yet. If I restart RS then it shows the new groups. 

Using Version
RetroShare Version: 0.6.5-1749-g120eb0263
Windows 10 Version 1909 QT 5.14.2

libretroshare
 - bzip2: 1.0.8, 13-Jul-2019
 - OpenSSL: OpenSSL 1.1.1g  21 Apr 2020
 - SQLite: 3.30.1
 - SQLCipher: 4.3.0 community
 - UPnP (MiniUPnP): 2.1.20190824
 - Zlib: 1.2.11

**Comment by NyvenZA** [comment]
If I right click on the node and select Edit Group, the Group Edit Dialog opens. 
This list also doesn't contain the new groups.

**Comment by NyvenZA** [comment]
My Retroshare crashed again witht he Distant search changed issue and then when i started up, all the group changes I have made was lost. 

The Groups i created were still there, but none of the nodes in the groups where there anymore. 

I opened the "Edit Groups" from the Right Click menu on an existing group and I could not see the see the groups there.

If I right click on an existing node and try to move it to another group, all my new groups are still listed there.
